sqlalchemy_risingwave/__init__.py

```diff
@@ -1,6 +1,6 @@
 from sqlalchemy.dialects import registry as _registry
 
-__version__ = "0.0.3"
```

Comment:
Is it true that we already reach 0.0.7?

Reply:
Emm, I push a version 0.0.7 to pypi to override 0.0.6. We have talked about the version in slack. So I change this version number.